CTP/sql/bin/run.sh

```diff
@@ -312,20 +325,48 @@ function config_cubrid_without_ha()
 
 }
 
-function config_cqt_tool()
+function config_qa_tool()
 {
      curDir=`pwd`
-     build_ver_type=""
-     qa_db_xml_path=${CTP_HOME}/sql/configuration/Function_Db/${db_name}_qa.xml
-     avaliable_broker_port=`awk '/SERVICE[[:space:]]*=[[:space:]]*ON/, /BROKER_PORT/' $CUBRID/conf/cubrid_broker.conf|grep BROKER_PORT|grep -v '#'|awk -F '=' '{print $2}'|head -1| tr -d ' '`
-     db_url="<dburl>jdbc:cubrid:localhost:${avaliable_broker_port}:${db_name}:::</dburl>"
-     sed -i "s#<dburl>.*</dburl>#$db_url#g" $qa_db_xml_path
-     if [ "$cubrid_bits" == "32" ];then
-          build_ver_type="32bits"
+     if [ "$interface_type" == "cci" ];then
```

Comment:
1. Move these codes to Makefile.

2. How to check whether support cci_set_cas_change_mode function:
cat $CUBRID/include/cas_cci.h | grep "cci_set_cas_change_mode"  | wc -l